app/decorators/valkyrie/resource_decorator.rb

```diff
@@ -1,5 +1,25 @@
 # frozen_string_literal: true
 class Valkyrie::ResourceDecorator < ApplicationDecorator
+  self.suppressed_attributes = [
```

Comment:
No, I will document this in the code base for clarity.  This an attempt to structure the following View logic from Plum:

https://github.com/pulibrary/plum/blob/master/app/views/hyrax/base/_attributes.html.erb#L8
https://github.com/pulibrary/plum/blob/master/app/schemas/plum_schema.rb#L302